testcafe/editResource.js

```diff
@@ -7,7 +7,19 @@ const editResourcePage = new EditResourcePage();
 const servicePage = new ServicePage();
 
 fixture`Edit Resource`
-  .page(ResourcePage.url(1));
+  .page(ResourcePage.url(1))
+  .afterEach(async t => {
+    const { log, warn, error } = await t.getBrowserConsoleMessages();
+
+    console.log('Captured console.log() messages:');
```

Comment:
Seems like (later) we could put these behind a conditional if `log`, `warn`, etc. is empty or null (since these appear to be arrays).

Reply:
Yeah, although unfortunately, our app constantly prints stuff to `console.log()` and `console.error()` even during normal behavior. For instance, every page load on a development server causes the "React-Hot-Loader: misconfiguration detected, using production version in non-production environment." error message. It looks like the right way to do this kind of thing in TestCafe is with a custom [Reporter](https://devexpress.github.io/testcafe/documentation/guides/extend-testcafe/reporter-plugin.html), so I might port this code into a reporter once I actually get it working.

I'm still trying to debug the issue, since I still apparently have not logged enough information to get the real answer.